src/test/scala/com/fulcrumgenomics/bam/pileup/PileupBuilderTest.scala

```diff
@@ -323,6 +343,24 @@ class PileupBuilderTest extends UnitSpec {
       piler.safelyClose()
     }
 
+    it should "exclude records that appear to be in an FR pair but are on different chromosomes" in {
```

Comment:
Yeah, this test is overly protective since it is for a read pair that "appears to be FR" based only on start/end positions and strand (but not chromosome) to always ensure reference sequence is taken into consideration. I added it to double-check some behavior while investigating and decided to keep it now that it is written.